File: javascript-closure-angular-client/API/Client/EntityInformerForGoods.js
goog.provide('API.Client.EntityInformerForGoods');

/**
 * @record
 */
API.Client.EntityInformerForGoods = function() {}

/**
 * @type {!API.Client.EntityType}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.type;

/**
 * @type {!API.Client.FederalTaxRegime}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.taxRegime;

/**
 * @type {!API.Client.EntityInformerForGoods_federalTaxRegime}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.federalTaxRegime;

/**
 * @type {!string}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.cityCode;

/**
 * @type {!API.Client.Address}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.address;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.details;

/**
 * @type {!boolean}
 * @export
 */
API.Client.EntityInformerForGoods.prototype.icmsTaxPayer;

File: async-scala-client/src/main/scala/io/swagger/client/model/EntityInformerForGoods.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class EntityInformerForGoods (
  _type: Option[EntityType],
taxRegime: Option[FederalTaxRegime],
federalTaxRegime: Option[EntityInformerForGoods_federalTaxRegime],
cityCode: Option[String],
address: Option[Address],
details: Option[Any],
icmsTaxPayer: Option[Boolean]
)
